server_installation/topics/operator/keycloak-cr.adoc

```diff
@@ -28,7 +28,10 @@ The Keycloak custom resource is a YAML file that defines the parameters for inst
 ifeval::[{project_community}==true]
  or an Ingress for Kubernetes
 endif::[]
- for the {project_name} cluster.
+ for the {project_name} cluster. You can set `host` to override automatically chosen host name for Route
```

Comment:
add "the" after "automatically"

Reply:
@andymunro Just want to double check. What we are trying to support is to allow overriding "automatically chosen" host name. Here is the snippet from Openshift Route type.

```
// host is an alias/DNS that points to the service. Optional.
	// If not specified a route name will typically be automatically
	// chosen.
	// Must follow DNS952 subdomain conventions.
	Host string `json:"host" protobuf:"bytes,1,opt,name=host"`
```